src/FlipMove.js

```diff
@@ -479,7 +479,11 @@ class FlipMove extends Component<void, ConvertedProps, FlipMoveState> {
       }
 
       elements.push(child);
-      domNodes.push(this.childrenData[childKey].domNode);
+
+      const childData = this.childrenData[childKey]
+      if (childData) {
+        domNodes.push(childData.domNode);
+      }
```

Comment:
For clarification, this is because there's a `prebuild` script that will run the linter, flow checker, and tests. It'll catch the missing semicolon that's causing CI to fail.

(Thanks again for adding that script, @Hypnosphi! And thanks for this review)